tests/src/Nightwatch/Commands/thunderScrollIntoView.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * @file
+ * Scroll element into center of the screen, so that it's visible and clickable.
+ *
+ * NOTE: This function works only with XPATH!!!
+ *
+ * This provides a custom command, .thunderScrollIntoView()
+ *
+ * @param {string} selector
+ *   The XPATH selector for element.
+ *
+ * @return {object}
+ *   The 'browser' object.
+ */
+
+/* eslint-disable func-names */
+exports.command = function thunderScrollIntoView(selector) {
```

Comment:
I think we can use https://nightwatchjs.org/api/moveToElement.html here instead - no?

Reply:
Problem with existing `move to element` functionalities is that they move an element into view of the browser window, but overlay elements are still there. So when the driver does click it clicks on overlay element and the test fails. That's why we are using scroll into view - what is actually doing scroll into the middle of the view.